lib/clif-backend/Cargo.toml

```diff
@@ -10,15 +10,15 @@ readme = "README.md"
 
 [dependencies]
 wasmer-runtime-core = { path = "../runtime-core", version = "0.7.0" }
-cranelift-native = { version = "0.31" }
-cranelift-codegen = { version = "0.31" }
-cranelift-entity = { version = "0.31" }
-cranelift-frontend = { package = "wasmer-clif-fork-frontend", version = "0.33" }
-cranelift-wasm = { package = "wasmer-clif-fork-wasm", version = "0.33" }
-target-lexicon = "0.4"
-wasmparser = "0.35.1"
-byteorder = "1.3"
-nix = "0.15"
+cranelift-native = { git = "https://github.com/wasmerio/cranelift", branch = "feature/merge-cranelift-0-43-1" }
+cranelift-codegen = { git = "https://github.com/wasmerio/cranelift", branch = "feature/merge-cranelift-0-43-1" }
+cranelift-entity = { git = "https://github.com/wasmerio/cranelift", branch = "feature/merge-cranelift-0-43-1" }
+cranelift-frontend = { git = "https://github.com/wasmerio/cranelift", branch = "feature/merge-cranelift-0-43-1" }
+cranelift-wasm = { git = "https://github.com/wasmerio/cranelift", branch = "feature/merge-cranelift-0-43-1" }
```

Comment:
TODO Update these before merging

Reply:
Question: it seems that `cranelift-entity` and `cranelift-codegen` custom forks are no longer needed? @bjfish

Reply:
Yes, also cust `cranelift-native` should not be needed. We will update these before merging.